src/com/projectkorra/rpg/worldevent/WorldEvent.java

```diff
@@ -119,23 +122,26 @@ public boolean getDarkenSky() {
 	public boolean getCreateFog() {
 		return createFog;
 	}
+	
+	@Override
+	public ChatColor getTextColor() {
+		return text;
+	}
+	
+	@Override
+	public BarColor getBarColor() {
+		return bar;
+	}
 
 	public static WorldEvent get(String name) {
 		return events.containsKey(name.toLowerCase()) ? events.get(name.toLowerCase()) : (ProjectKorraRPG.getFileManager().loadFile(name.toLowerCase()) == null ? null : get(name));
 	}
 
-	public static Set<WorldEvent> getEventsByElement(Element e) {
-		if (!eventsByElement.containsKey(e)) {
-			eventsByElement.put(e, new HashSet<>());
-		}
-		return eventsByElement.get(e);
-	}
-
 	public static Set<String> getEventNames() {
-		return events.keySet();
+		return new HashSet<>(events.keySet());
 	}
 
-	public static Collection<WorldEvent> getEvents() {
-		return events.values();
+	public static Set<WorldEvent> getEvents() {
+		return new HashSet<>(events.values());
```

Comment:
Is there a reason you used a general HashSet rather than a typed one?

Reply:
Inferred typing